SimplenoteIntents/IntentHandler.swift

```diff
@@ -1,23 +1,44 @@
 import Intents
+import CoreData
 
 class IntentHandler: INExtension {
+    private var dataManager: CoreDataManager = {
+        let storageSettings = StorageSettings()
+        let coreDataManager = CoreDataManager(storageSettings: storageSettings)
+        coreDataManager.managedObjectContext.persistentStoreCoordinator = coreDataManager.persistentStoreCoordinator
+
+        return coreDataManager
+    }()
```

Comment:
A note: we need to be careful here. If migration in the main app failed, this code will probably create a new empty database in the shared container? And this will make the app to "loose" it's data once it sees that there is a database in the shared container.

Reply:
+1'ing to what Evgeny mentioned. We need to handle the "Non cconfigured State". That is:

1. Safeguard against the DB not existing (otherwise spawning the CoreDataManager would create the DB!)
2. Probably display a placeholder in the UI, saying "please log into your Simplenote account first"